integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/AnsibleResource.java

```diff
@@ -0,0 +1,72 @@
+package com.redhat.service.smartevents.integration.tests.resources;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+
+public class AnsibleResource {
+
+    private static final String ANSIBLE_ENDPOINT = System.getProperty("ansible.endpoint") + "/api/v2";
+    private static final String ANSIBLE_USERNAME = System.getProperty("ansible.username");
+    private static final String ANSIBLE_PASSWORD = System.getProperty("ansible.password");
+
+    public static List<String> jobTemplatesList(String name) {
+        final List<String> templateNames = given()
+                .baseUri(ANSIBLE_ENDPOINT)
+                .auth().preemptive().basic(ANSIBLE_USERNAME, ANSIBLE_PASSWORD)
+                .relaxedHTTPSValidation()
+                .basePath("job_templates")
+                .when()
+                .get()
+                .then()
+                .statusCode(200)
+                .extract()
+                .jsonPath()
+                .getJsonObject("results.name");
+
+        return templateNames;
+    }
+
+    public static int jobTemplateID(String name) {
+        final int templateID = given()
+                .baseUri(ANSIBLE_ENDPOINT)
+                .auth().preemptive().basic(ANSIBLE_USERNAME, ANSIBLE_PASSWORD)
+                .relaxedHTTPSValidation()
+                .basePath("job_templates")
+                .queryParam("name", name)
+                .when()
+                .get()
+                .then()
+                .statusCode(200)
+                .extract()
+                .jsonPath()
+                .getInt("results.id[0]");
+
+        return templateID;
+    }
+
+    public static String jobs(int jobTemplateId, int days, int maxItems) {
```

Comment:
What does this method return?
From the method name it looks like it should return some list, however the return type is String.